### PR TITLE
docs: add cypress-intercept-formdata to plugins.json

### PR DESCRIPTION
**Pull request by yoavniran**
- [x] 🚀 Works with the latest major version of Cypress
- [x] 🛠 Plugin purpose is clearly documented ([readme](https://github.com/yoavniran/cypress-intercept-formdata#readme))
- [x] 📝 Well-written documentation ([usage](https://github.com/yoavniran/cypress-intercept-formdata#usage))
- [x] 🔬 Tested using Cypress - tests using Cypress can act as both example usage and test coverage ([report](https://github.com/yoavniran/cypress-intercept-formdata/actions/runs/6377472389/job/17306259604))
- [x] 👷‍♀️ CI pipeline that's passing ([GH actions](https://github.com/yoavniran/cypress-intercept-formdata/actions))

### CODE REVIEW

File: src/data/plugins.json
Comment (MikeMcC399) [inline src/data/plugins.json:744]:
```suggestion
          "description": "Work with Cypress' Intercept multipart/form-data requests",
```

The same typo is also on the website description https://github.com/yoavniran/cypress-intercept-formdata in the About section.

Comment (yoavniran) [reply src/data/plugins.json]: thanks! will fix in repo too

**Review by MikeMcC399 — APPROVED** [review]

**Comment by yoavniran** [comment]
hey @MikeMcC399 
anything I need to do to help get this merged?
thanks

**Comment by MikeMcC399** [comment]
Hi @yoavniran 

> anything I need to do to help get this merged? thanks

You could make sure that your plugin fits the checklist. Copy and paste this into your original post.

Before submitting your plugin, please check to see if it fulfills these requirements:

- [ ] 🚀 Works with the latest major version of Cypress
- [ ] 🛠 Plugin purpose is clearly documented (in a README or docs website)
- [ ] 📝 Well-written documentation - A great example ([link](https://github.com/Fredx87/cypress-keycloak-commands#cypress-keycloak-commands))
- [ ] 🔬 Tested using Cypress - tests using Cypress can act as both example usage _and_ test coverage
- [ ] 👷‍♀️ CI pipeline that's passing (CircleCI and [Github Actions](https://github.com/cypress-io/cypress-tutorial-build-todo/blob/main/.github/workflows) are both free for Open Source)

If you have reason to believe your plugin does not need to meet certain requirements, please feel free to provide justification below:

Other than that, I'm only a community contributor. You would need to get approval from a Cypress member for merger. In fact I'm also waiting for several PRs, which I submitted, to be merged. It seems that Cypress changed their organization last week, so I don't know how this is going to be worked now.

**Review by mschile — APPROVED** [review]

**Comment by mschile** [comment]
Hi @yoavniran 👋, thanks for the plugin contribution!

**Pull request merged**
